lib/iris/util.py

```diff
@@ -1554,3 +1554,104 @@ def _meshgrid(*xi, **kwargs):
         if mxii.dtype != xii.dtype:
             mxi[i] = mxii.astype(xii.dtype)
     return mxi
+
+
+def find_discontiguities(cube, rel_tol=1e-5, abs_tol=1e-8):
+    """
+    Searches coord for discontiguities in the bounds array, returned as a
+    boolean array (True where discontiguities are present).
+
+    Args:
+        cube:
```

Comment:
Sorry to be a pain, but can you reshape these docstrings a bit, to get them in the approved format, more like other things in this module for example ?
So e.g. after a bit of massage, this one might read :
```
    Args:
        cube (`iris.cube.Cube`):
            The cube to be checked for discontinuities in its 'x' and 'y' coordinates.

    Kwargs:
        rel_tol (float):
            The relative equality tolerance to apply in coordinate bounds checking.
        abs_tol (float):
            The absolute value tolerance to apply in coordinate bounds checking.

    Returns:
        result (`numpy.ndarray` of bool) :
            true/false map of which cells in the cube XY grid have discontiguities in the coordinate points array.
            This can be used as the input array for :func:`iris.util.mask_discontiguities`.

```

Reply:
Oh wow, since when did we write our docstrings like this?  I like them organised this way.

Reply:
Well it's set out in the developer guide https://scitools.org.uk/iris/docs/latest/developers_guide/documenting/docstrings.html#multi-line-docstrings

I believe "most" of Iris follows it, but there are certainly some exceptions in older code.